svthalia/Reaxit
# Fix fab button style

Closes #447.

### Summary
A clear and concise description of the changes that you made. 

### How to test
Steps to test the changes you made:
1. Go to calendar
2. Check the style
